### PR TITLE
Short Line Descriptions that Change as Code Runs

### PR DESCRIPTION
**Pull request by equationcrunchor**
Current descriptions are just placeholders mostly copy and pasted from the main descriptions, and need revision.

### CODE REVIEW

File: djweb/simulations/static/js/assembly_addition.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 const computer_size = 7;
 const num_code_lines = 4;
 const highlighted_registers = [6];
+const line_descriptions = [
+    "<b>CLA 4: </b> Clears the accumulator and stores the value at address 4 (which you'll see if you mouseover is 12) into the accumulator.<br />",
+    "<b>ADD 5: </b> Adds the value at address 5 (30) to the accumulator.<br />",
+    "<b>STO 6: </b> Stores the value of the accumulator into the register at address 6.<br />",
+    "<b>HTR: </b> Tells the computer to stop.<br />",
+];
```
Comment (srisi) [inline djweb/simulations/static/js/assembly_addition.js]: From below (I don't know how to properly access it):
```
    computer.general_memory[4].fixed_point = 12;
    computer.general_memory[5].fixed_point = 30;
```
Why aren't these lines part of the program? Right now, when I press clear and assemble, memory slots 4 and 5 become 0.

File: djweb/simulations/static/js/assembly_addition.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 const computer_size = 7;
 const num_code_lines = 4;
 const highlighted_registers = [6];
+const line_descriptions = [
```
Comment (srisi) [inline djweb/simulations/static/js/assembly_addition.js]: I really appreciate this addition. However, I think the current organization structure is too complicated. 
Would it be possible to combine both the symbolic code and the descriptions into a Program class? 
Within the program class, each line could have both code, descriptions, and possibly highlight instructions (e.g. when executing a line that deals with the accumulator, adding an instruction to highlight the accumulator).

File: djweb/dj_comp_hist/migrations/0001_initial.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1 on 2018-12-07 21:36
+# Generated by Django 2.1.5 on 2019-03-18 18:14
```
Comment (srisi) [inline djweb/dj_comp_hist/migrations/0001_initial.py]: We're still deciding on what to do with the migrations (i.e. whether to include these). We'll report back in 1-2 hours.

File: djweb/simulations/static/js/assembly_addition.js (hunk 1 above)
Comment (equationcrunchor) [reply djweb/simulations/static/js/assembly_addition.js]: This demo was written before you could define numerical constants in the assembly code; now I'm wondering if we should just have this demo hardcode the values into the computer before introducing DEC and ORG in the floating point demo.  @alanabreu106 what do you think?

File: djweb/simulations/static/js/assembly_addition.js (hunk 2 above)
Comment (equationcrunchor) [reply djweb/simulations/static/js/assembly_addition.js]: That sounds useful.  @felixtran39 is working on changing the interface right now, maybe you guys should talk and we can put in a future pull request?  My concern is that Felix and I both changing the GUI at the same time could be problematic.

**Comment by ryaanahmed** [comment]
This PR is now obviated by https://github.com/dhmit/computation_hist/pull/230, which incorporates much of what @srisi suggested. Closing here.

**Pull request closed without merging**